CSS_files/components/cards/cards.css

```diff
@@ -0,0 +1,121 @@
+
+.card-content{
+    display: flex;
+    flex-direction: column;
+    
+}
+
+.card img{
+ 
+    object-fit: cover;
+    max-height: 20rem;
+}
+
+.card-content:hover{
+    background-color: whitesmoke;
+    border-radius: 10px;
+    
+}
+.card{
+    display:flex;
+    flex-direction: column;
+    width: 25rem;
+    background-color: var(--pure-white);
+    border-radius: 10px;
+}
+
+.card-img-top-br{
+    border-top-right-radius: 1rem;
+    border-top-left-radius: 1rem;
+}
+
+.card-heading,.card-text{
+    padding: .8rem 1rem;
+}
+
+.card-text:hover{
+    background-color: whitesmoke;
+}
+
+
+.card-footer{
+    display:flex;
+    padding: .5rem .8rem;
+    margin-top: none;
+
+}
+
+.card-footer i{
```

Comment:
Same as above . Use a class called as card-footer-icon and pass it to the icon element.